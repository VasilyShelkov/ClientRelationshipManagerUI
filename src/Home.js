import React from 'react';

export default ({ pattern, pathname }) => (
  <div>
    <h1>Router Home</h1>
    <ul>
      <li><strong>Pattern</strong>: {pattern}</li>
      <li><strong>Pathname</strong>: {pathname}</li>
    </ul>
  </div>
 );
