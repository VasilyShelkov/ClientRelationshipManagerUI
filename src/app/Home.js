import React from 'react';

export default ({ pattern, pathname }) => (
  <div>
    <h1>PLACEHOLDER HOME PAGE</h1>
    <h2>Super awesome cool homepage coming soon...</h2>
  </div>
 );
