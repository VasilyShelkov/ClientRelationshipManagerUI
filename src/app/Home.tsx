import * as React from 'react';

export default () => (
  <div>
    <h1>PLACEHOLDER HOME PAGE</h1>
    <h2>Super awesome cool homepage coming soon...</h2>
  </div>
);
