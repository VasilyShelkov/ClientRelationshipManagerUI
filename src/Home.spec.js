import React from 'react';
import renderer from 'react-test-renderer';
import Home from './Home';

describe('src/Home.js', () => {
  it('renders the Home page', () => {
    expect(renderer.create(<Home />).toJSON())
      .toMatchSnapshot();
  });
});
