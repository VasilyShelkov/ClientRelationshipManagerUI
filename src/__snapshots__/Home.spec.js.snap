exports[`src/Home.js renders the Home page 1`] = `
<div>
  <h1>
    Router Home
  </h1>
  <ul>
    <li>
      <strong>
        Pattern
      </strong>
      : 
    </li>
    <li>
      <strong>
        Pathname
      </strong>
      : 
    </li>
  </ul>
</div>
`;
